packages/jaeger-ui/src/components/TracePage/index.test.js

```diff
@@ -369,10 +347,18 @@ describe('mapDispatchToProps()', () => {
 });
 
 describe('mapStateToProps()', () => {
-  it('maps state to props correctly', () => {
-    const id = 'abc';
-    const trace = {};
-    const state = {
+  const id = 'abc';
+  const trace = {};
+  const embedded = Math.random();
```

Comment:
I mentioned this below as well, why use `Math.random()`, an equally valid, but easier to debug test value could be `const embedded = 'truthy embedded value';`